app/views/shared/_directory.html.erb

```diff
@@ -8,4 +8,11 @@
     <p class="directory__description"><%= item.description %></p>
   </div>
   <% end %>
+  <div class="directory__item">
+    <%= heading_tag(level: heading_level ||= 3, class: 'directory__title') do %>
+      <%= link_to t("home.show.blog_title"), 'https://blog.moneyadviceservice.org', class: 'directory__link' %>
```

Comment:
I know this code will probably have a short lifetime, but is there a reason why this uses a plain old URL here and going to the locales [here](https://github.com/moneyadviceservice/frontend/pull/875/files#diff-9c9d02deb56445d7f85f9bc4c16c9a0eR38)?

(Since it's not a localized URL, I'd tend to just use a straight URL everywhere.)